### ISSUE TITLE
[Bug] While the view is moving, any scroll behavior causes the nav bar to re-appear

### ISSUE THREAD

**Issue by plof27** [mozilla-mobile/fenix#20072]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
1. Dismiss the nav bar at the top by swiping up
2. Swipe your finger up to scroll the page downward
3. Swipe up again (to continue scrolling or modify speed)

(The same behavior is found when swiping in other directions)

### Expected behavior
The top nav bar stays hidden (unless swiping down)

### Actual behavior
The nav bar reappears, causing the whole page to shake up and down slightly

[Video demo](https://drive.google.com/file/d/1lXZYCxxS-AnkuCVrG3I9BFDCSmiKryvd/view?usp=sharing)

### Device information

* Device vendor / model and Android version: Google Pixel 2, Android 11
* Firefox for Android version: 89.1.1 (Build #2015812947)

**Comment by SoftVision-LorandJanos** [comment]
Hi @plof27 ! Thank you for your report! I am able to reproduce the issue you described, and I remember that a similar issue was logged before, I can't find it, however [this](https://github.com/mozilla-mobile/fenix/issues/16214) is somehow related but it was fixed.
Tested on the latest Nightly (91.0a1 Friday 6/18 @ 5:07 AM) and on Release 89.1.1 (Build #2015812947).
Devices used: Motorola Nexus 6 (Android 7.1.1), Samsung Galaxy Tab 3 (Android 9), Google Pixel 4 (Android 11).

**Comment by Mugurell** [comment]
I don't know of any recent changes around this in AC but it's happening in RB and not on GVE so it seems to be a recent regression there.

**Comment by Lann094** [comment]
Could we get an update on this? Extremely irritating when browsing.

**Comment by Susko3** [comment]
This issue is very similar to #19853. The difference is that this issue describes problems with swiping, and the other one with tapping. Fixing one issue will probably close the other, so please check both once a fix is in place.

**Comment by Mugurell** [comment]
I think this is fixed in the current Nightly after https://github.com/mozilla-mobile/android-components/pull/10562
@plof27 Can you confirm?

**Comment by plof27** [comment]
> I think this is fixed in the current Nightly after [mozilla-mobile/android-components#10562](https://github.com/mozilla-mobile/android-components/pull/10562)
> @plof27 Can you confirm?

Can confirm! I downloaded the latest nightly on my phone and tablet, and the issue went away! Thanks 👍

**Comment by SoftVision-LorandJanos** [comment]
This issue is no longer reproducible on the latest Nightly 98.0a1 (2022-02-02).
Tested using an Oppo Reno 6 (Android 11).
Closing the ticket as fixed.
